### ISSUE TITLE
nodes disconnecting after a few minutes

### ISSUE THREAD

**Issue by samuec2switch** [mysteriumnetwork/node#5227]
i'm a consumer , i have around 180 simultaneous connection to different nodes. in the last 4 days me and some friends noticed a big disconnection issue , more than half of the connections disconnects after a few minutes you connect and the bigger problem is the "domino effect" i connect to 5 nodes? 10 more dies , i connect to 10? 15 more dies , all the times and it's very anoying. this issue started after the hermes update. maybe it's related to it? no matter to which node i'm connected to and in which country , the problem persists. not all of my 180 connections are affected by this issue , the ones that works never disconnected from the beginning or if they disconnect is very very rare compared to some others , before the update almost none of the nodes i'm connected to ever died or disconnected , some of them was even running from 80+ hours.

**Comment by adinetech** [comment]
Clear Explanation: A user is running multiple sessions using multiple MysteriumVPN instances, a total of 180 connections to random Residential nodes from the single IP and after the Hermes migration whenever the user connects to 5 nodes then 10 nodes will get disconnected automatically, and eventually most of the sessions will disconnect.

Logs, which tells the connection disconnects.

```2022-06-20T08:28:28.162 ERR ../../core/connection/manager.go:933           > Failed to send p2p keepalive ping. SessionID=3344d23a-72f8-414b-8aa6-1ee0cc8e3beb error="timeout waiting for reply to \"p2p-keepalive\": p2p send timeout"
2022-06-20T08:28:28.953 WRN ../../p2p/channel.go:395                       > Stream 1451 not found, message data:
2022-06-20T08:28:28.954 WRN ../../p2p/channel.go:395                       > Stream 1450 not found, message data:
2022-06-20T08:28:38.164 ERR ../../core/connection/manager.go:933           > Failed to send p2p keepalive ping. SessionID=3344d23a-72f8-414b-8aa6-1ee0cc8e3beb error="timeout waiting for reply to \"p2p-keepalive\": p2p send timeout"
2022-06-20T08:28:38.164 ERR ../../core/connection/manager.go:936           > Max p2p keepalive err count reached, disconnecting. SessionID=3344d23a-72f8-414b-8aa6-1ee0cc8e3beb
2022-06-20T08:28:38.164 INF ../../core/connection/manager.go:715           > Connection state: Connected -> Disconnecting```

**Comment by adinetech** [comment]
Requested the user to use the latest node version as consumer and get nodes from https://discovery.mysterium.network/api/v3/proposals?ip_type=residential&compatibility_min=2

It seems like fixed for now, and 1.9.1 need to be released in the node launchpad.
